modules/oc-mirror-disk-to-mirror-v2.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="disk-mirror-v2_{context}"]
+== Mirroring from disk to mirror
+
+.Procedure
+
+* Process the image set file on the disk and mirror the contents to a target mirror registry by running the following command:
++
+[source,terminal]
+----
+$ oc mirror -c isc.yaml --from file://<file_path> docker://<mirror_registry_url> --v2 <1>
+----
+<1> Specify the URL or address of the mirror registry where the images are stored and from which they need to be deleted.
+
+.Verification
+
+* Navigate to the `cluster-resources` directory within the `working-dir` directory that was generated at `<file_path>.
```

Comment:
Missing a backtick and same comment about using the wording you did at https://github.com/openshift/openshift-docs/pull/77247/files#diff-67f7c190e8a0b66d7729524d1815ec060ebbc1f5450dcd9a58b10d8976779024R27